tutorials/introduction-to-jsosolvers/index.jmd

```diff
@@ -58,13 +58,46 @@ nlp.counters
 ```julia
 F(x) = [x[1] - 1; 2 * (x[2] - x[1]^2)]
 nls = ADNLSModel(F, [-1.2; 1.0], 2)
-trunk(nls, verbose = 1)
+trunk(nls, atol = 1e-6, rtol = 1e-6)
 ```
 
 ```julia
 nls.counters
 ```
 
+We conclude these examples by a simple nonlinear regression.
+
+We build a nonlinear model `m` with unknown parameters α and β, and generate randomly some observations of this model.
+```julia
+m(α, β, x) = α * cos(x) + β * sin(x) # nonlinear models with unknown α and β
+
+ndata = 100 # size of data sample
+data = [rand() * 2 - 1 for i=1:ndata]
+obs = [m(0.5, 0.5, xi) for xi in data] + rand(ndata) / 10; # observations
```

Comment:
Here is the data: https://gist.github.com/abelsiqueira/8ca109888b22b6ab1e76825f0567c668
```suggestion
url_prefix = "https://gist.githubusercontent.com/abelsiqueira/8ca109888b22b6ab1e76825f0567c668/raw/f3f38d61f750b443fb4307efbf853447275441a5/"
data = CSV.read(download(joinpath(url_prefix, "thurber.csv")), DataFrame)
x, y = data.x, data.y
```